### PR TITLE
Nima JsonB Stream

### PR DESCRIPTION
**Pull request by SentryMan**
Now for smaller payloads, helidon nima now will not duplicate the byte array (the makes jsonb the first json serializer to not auto copy the buffer for short requests)

### CODE REVIEW

**Comment by SentryMan** [comment]
hmm, forgot that nima only compiles to 19

File: http-api/src/main/java/io/avaje/http/api/NimaJsonbStream.java [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+  @Override
+  public void write(int b) throws IOException {}
+
+  @Override
+  public void write(byte[] b, int off, int len) throws IOException {
+    if (chunked) {
+      outputStream.write(b, off, len);
+    } else if (firstBuffer == null) {
+      firstBuffer = b;
```
Comment (rbygrave) [inline http-api/src/main/java/io/avaje/http/api/NimaJsonbStream.java:29]: I think there is a subtle issue here in that the buffer will get reused / overwritten (if there are 2 or more "buffer writes" / if content exceeds 4k).

That is, I don't think we can keep a reference to b, off, len ... in the form of firstBuffer etc. We'd need to make a defensive copy as we don't know if this was the "first and last write" or "just the first or many writes".

**Comment by SentryMan** [comment]
Now I use the new Jsonb stuff

**Comment by SentryMan** [comment]
ah woops, forgot to bump the jsonb versions

**Pull request merged**